src/content/docs/developer-tools/kinde-api/api-example-node.mdx

```diff
@@ -11,7 +11,7 @@ relatedArticles:
 
 If you don’t use Postman or other similar tools to test your connections, here’s an example of how to call the [Kinde Management API](https://kinde.com/api/docs/#kinde-management-api) in NodeJS.
 
-1. [Add a machine to machine application](/developer-tools/kinde-api/add-a-m2m-application-for-api-access/) for API access, and copy the **App keys - Domain, Client ID** and **Client secret** from the application.
+1. [Add a machine to machine application](/developer-tools/kinde-api/about-m2m-scopes/) for API access, and copy the **App keys - Domain, Client ID** and **Client secret** from the application.
```

Comment:
@marcosmartini I'm not sure these link updates are correct as they all seem to be now pointing at the scopes page?

Reply:
I will fix